cb_medical_views/views/medical_patient_views.xml

```diff
@@ -7,9 +7,21 @@
             <search>
                 <field string="Name" name="name" />
                 <field string="VAT" name="vat" />
+                <field string="Birth date" name="birth_date" />
             </search>
         </field>
     </record>
+    <record id="medical_patient_tree" model="ir.ui.view">
+        <field name="name">medical.patient.tree</field>
+        <field name="model">medical.patient</field>
+        <field name="inherit_id" ref="medical_administration.medical_patient_tree" />
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="vat" string='NIF' />
```

Comment:
do not translate this way, transaltion should be managed in other ways

Reply:
you're right, thx